docs/journal/01-cold-storage.md

```diff
@@ -3,66 +3,213 @@ _Capturing event data and saving it in its raw format to cold storage_
 
 :pencil2: working draft
 
-## Fabrikam's perspective
+> The purpose of this phase is to determine an approach to ingesting data, performing the simplest of processing, and saving it for subsequent analysis. Because the customer requires all event data to be stored indefinitely, the volume of data held in cold storage could become very large. Cold storage must therefore be inexpensive.
+
+# Fabrikam's Perspective
 Fabrikam's engineering team has very little experience with high-scale event-oriented systems. They want to make sure that they are [asking the right questions][orientation]. They also need to get a minimum viable product deployed.
 
 For their [first milestone][milestone] they want to:
 - Ingest a representative set of simulated events.
 - Store the raw data for all events for later analysis. [:arrow_right:][cold-storage]
-- Increase the scale until they match the targets for there first contract [:arrow_right:][increase-scale]
+- Increase the scale until they match the targets for their customer [:arrow_right:][increase-scale]
```

Comment:
Odd. It renders here: https://github.com/mspnp/iot-journey/blob/master/docs/journal/01-first-steps.md
Perhaps this is just a bug in the rendering of the pr? :confused: